lib/google/drive.ts

```diff
@@ -45,6 +45,22 @@ export default class Drive {
     })
     return searchResults.data ?? { files: [] }
   }
+
+  //search all folders from shared drive
+  async getAllDirectories() {
```

Comment:
could just loop with pagination? not sure if its smart to have such large array in state?